src/ucs/stats/serialization.c

```diff
@@ -75,13 +75,13 @@ SGLIB_DEFINE_HASHED_CONTAINER_FUNCTIONS(ucs_stats_clsid_t, UCS_STATS_CLS_HASH_SI
 #define FREAD(_buf, _size, _stream) \
     { \
         size_t _nread = fread(_buf, 1, _size, _stream); \
-        assert(_nread == _size); \
+        ucs_assert(_nread == _size); \
```

Comment:
are there other places in the code that use assert()?

Reply:
We have. Also libs that we use (`khash`, `sglib`) use `assert` as well.
Assertion in `test_ucp.c` and `test_uct.c` may cause the same problem if compiled with `DNDEBUG`.
For the rest in UCX code, apparently the tested expression is in use so it's not a problem.